spyder/plugins/ipythonconsole.py

```diff
@@ -821,13 +821,35 @@ def get_plugin_actions(self):
         """Return a list of actions related to plugin."""
         create_client_action = create_action(
                                    self,
-                                   _("Open an &IPython console"),
+                                   _("Open a new console"),

```

Comment:
Let's drop the `Open` here.

Reply:
And ``a``, of course, if you're going to do that.

Reply:
how about:

`New console (default settings)`

Reply:
How about just ``New console (default)`` as discussed above.

Reply:
I agree with @CAM-Gerlach on this one.